### DISCUSSION TITLE
How to generate Swagger Doc for Hasura generated REST endpoint.

### DISCUSSION THREAD

**Discussion by edmundmunday** [hasura/graphql-engine#7751] [Question] [answered]
Following the instructions here: https://hasura.io/docs/latest/graphql/core/api-reference/restified.html#openapi-3-specification

When calling the URL specified in the docs, I get the following error response: 
```
{
  "path": "$",
  "error": "resource does not exist",
  "code": "not-found"
}
```

I've asked in the discord and nobody seems to have any suggestions.

**Comment by tirumaraiselvan** [comment, marked as answer]
@edmundmunday It appears the docs is ahead of the actual release. The endpoint will be available in v2.1.0-beta.2 (next release in few days)

**Reply by edmundmunday** [reply]
Thanks @tirumaraiselvan - was wondering if that might be the case.

**Reply by disambiguist** [reply]
Thanks!

**Reply by jwarwick-delfi** [reply]
Hi @tirumaraiselvan  - I'm using Hasura cloud and still getting 404s against `/api/swagger/json` on version v2.3.1-cloud.1. Can you confirm that it has been deployed?

**Reply by tirumaraiselvan** [reply]
> Hi @tirumaraiselvan - I'm using Hasura cloud and still getting 404s against `/api/swagger/json` on version v2.3.1-cloud.1. Can you confirm that it has been deployed?

Sorry about that. We are working on making this available on Cloud within a week.

**Reply by jwarwick-delfi** [reply]
I can work with that, thanks!
